transitive_maven_jar/BUILD

```diff
@@ -0,0 +1,4 @@
+package(
```

Comment:
Remove the content, leave an empty file.  .bzl files don't need to be marked public.